doc/conf.py

```diff
@@ -0,0 +1,32 @@
+project = 'echemdb'
+copyright = '2022, the echemdb website authors'
+author = 'the echemdb authors'
+
+release = '0.0.0'
+
+
+extensions = ["sphinx.ext.autodoc", "sphinx.ext.todo", "myst_nb"]
+
+source_suffix = {
+    '.rst': 'restructuredtext',
+    '.ipynb': 'myst-nb',
+    '.myst': 'myst-nb',
+}
+
+templates_path = ['_templates']
+
+exclude_patterns = ['generated', 'Thumbs.db', '.DS_Store', 'README.md', 'news', '.ipynb_checkpoints', '*.ipynb', '**/*.ipynb']
+
+todo_include_todos = True
+
+html_theme = 'sphinx_rtd_theme'
+
+html_static_path = []
+
+# Add Edit on GitHub links
+# html_context = {
+#     'display_github': True,
+#     'github_user': 'echemdb',
+#     'github_repo': 'website',
+#     'github_version': 'main/doc/',
+# }
```

Comment:
Linkchecker fails, when new doc files are created, which are not available in the published repo. Hence all "Edit on Github pages" Link checks fail.

I did not find a workaround other than turning it off in any PR that contains new sites in the docu section.

Reply:
You can tell linkchecker to ignore certain URLs. Maybe we should just ignore the ones that go to the edit on github targets?